app/controllers/user_controller.rb

```diff
@@ -4,6 +4,7 @@ def index
   end
 
   def show
-    puts params
+    @users = User.all
+    @user = User.find(params[:id])
```

Comment:
- [ ] The **User_controller** should  have a `def current_user`.

Reply:
Done